HomeUI/src/views/apps/Management.vue

```diff
@@ -4579,7 +4576,7 @@ export default {
       this.possibleLocations = possibleLocations;
     },
     continentsOptions(isNegative) {
-      const continents = [{ value: isNegative ? 'NONE' : 'ALL', text: isNegative ? 'NONE' : 'ALL' }];
+      const continents = [{ value: isNegative ? 'ALL' : 'ALL', text: isNegative ? 'ALL' : 'ALL' }];
```

Comment:
I think this was ok, on forbidden, if none is selected on continent it should say None not all. As it will not set anything if the it's on the default choice.